CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# CONTRIBUTING
+
+This guide is for anyone who wants to contribute to the PetCentral project.
+
+## Working on an issue
+
+IMPORTANT: Before working on a PR, please open an issue and discuss your intended changes. Maintainers may be able to provide valuable information and point you in the right direction to get the most out of your contribution. If you send a PR totally out of the blue there is a chance that it will not get accepted.
+
+### Branches
+
+All `issue` branches should be branched from `main` branch unless you're working on an issue that belongs to one of our special projects. In that
+case, a project branch will be specified. If you're unsure about the branch don't hesitate to reach out.
+
+### Workflow
+
+- create an issue branch
+- commit your changes
+- open a PR
```

Comment:
This is so thorough and well done!  Curious RE: the few odd line breaks.  Otherwise this all looks great.

Reply:
Thanks for the feedback Mallory! In the CONTRIBUTING.md file for lines 16-20 those line breaks are bullet points. It should look something like this when viewed on GitHub

![Contributing](https://github.com/SA-BlueOcean/PetCentral/assets/69211773/f9f22f81-8951-4941-9c3f-7c7486836958)

I'm not at all against changing from bullet points to an alternative, so just let me know if there's another format or way that we should be listing those steps out and I'll make it happen.

Reply:
It looks great!  No need to change at all.  Love it!